src/common/utils/__tests__/multiYearPlanHelperFunctions.test.ts

```diff
@@ -0,0 +1,75 @@
+import { deepStrictEqual, strictEqual } from 'assert';
+import {
+  NUM_SEMESTERS,
+  NUM_YEARS,
+  TERM,
+} from 'common/constants';
+import { metadata } from 'testData';
+import flatten from 'lodash.flatten';
+import { stub } from 'sinon';
+import * as multiYearPlan from '../multiYearPlanHelperFunctions';
+import { calculateSemesters } from '../multiYearPlanHelperFunctions';
+
+describe('Multi-Year Plan Helper Functions', function () {
+  describe('calculateSemesters', function () {
+    it('returns the requested number of semesters', function () {
+      const actual = calculateSemesters(
+        metadata.currentAcademicYear, NUM_SEMESTERS
+      );
+      strictEqual(actual.length, NUM_SEMESTERS);
+    });
+    it('returns the correct terms', function () {
+      const years = Array(NUM_YEARS);
+      const terms = [TERM.FALL, TERM.SPRING];
+      const expectedTerms = flatten(years.fill(terms));
```

Comment:
Along the lines of my previous note about `lodash`, there are a couple of workarounds for [flattening a single-level array[(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#reduce_and_concat), the easiest being:

```ts
const expectedTerms = [].concat(...years.fill(terms));
```

We're also going to need to upgrade to Node v14 soon anyway, at which point we can use `Array.flat()` and `Array.flatMap()` out of the box

Reply:
Ah, didn't know about this. Thanks! This was updated in e2dea8e.